thirdeye/thirdeye-frontend/app/pods/manage/alerts/template.hbs

```diff
@@ -0,0 +1,144 @@
+<aside class="manage-alert-filters col-md-3">
```

Comment:
make sure to wrap all your col-*-* with a bootstrap row. 
The container adds negative -15px margin left and right, the row offsets those.